acceptance-tests/apps/gcpobjectstorage/src/main/java/broadcom/spring/gcp/object/storage/gcpobjectstorage/GcpobjectstorageApplication.java

```diff
@@ -0,0 +1,13 @@
+package broadcom.spring.gcp.object.storage.gcpobjectstorage;
```

Comment:
As this project is owned by the CF foundation, it's probably better to use `org.cloudfoundry...` as the package name